src/types/character/character-type.ts

```diff
@@ -0,0 +1,17 @@
+import { Stat } from './stat-type'
+import { Item } from '../shop/item-type'
+import { Effect } from '../skills/effect-type'
+
+export type CharacterType = {
```

Comment:
Like interfaces, adding type as suffix is bad.
Because interfaces and types are almost the same stuff except a type should be an alias or a merge of multiple interface and primitive types.
So the right thing to do here is:
- change type to interface
- remove type as suffix
- define if you want to use a convention when naming your types and interfaces and this convention should be the same one for both (ex: add model as suffix, add I as prefix, do nothing, etc...) I use I as prefix on my hand.

Reply:
Will do!